File: CRS-Group-A-Java-Spring-REST/src/main/java/com/lti/bean/Payment.java
package com.lti.bean;

import java.time.LocalDate;
import java.time.LocalDateTime;

import org.springframework.stereotype.Component;

/**
 * @author Sebastian
 *
 */

@Component
public class Payment {
	
	private int paymentId;
	private int studentId;
	private int paymentAmount;
	private LocalDate dueDate;
	private String semester;
	private int isPaid;
	private String paymentMethod;
	
	public Payment(int studentId, int paymentAmount, LocalDate dueDate, String semester) {

		this.studentId = studentId;
		this.paymentAmount = paymentAmount;
		this.dueDate = dueDate;
		this.semester = semester;
	}
	
	/**
	 * @return the isPaid
	 */
	public int getIsPaid() {
		return isPaid;
	}

	/**
	 * @param isPaid the isPaid to set
	 */
	public void setIsPaid(int isPaid) {
		this.isPaid = isPaid;
	}

	/**
	 * @return the paymentMethod
	 */
	public String getPaymentMethod() {
		return paymentMethod;
	}

	/**
	 * @param paymentMethod the paymentMethod to set
	 */
	public void setPaymentMethod(String paymentMethod) {
		this.paymentMethod = paymentMethod;
	}

	/**
	 * @return the paymentId
	 */
	public int getPaymentId() {
		return paymentId;
	}

	/**
	 * @param paymentId the paymentId to set
	 */
	public void setPaymentId(int paymentId) {
		this.paymentId = paymentId;
	}

	/**
	 * @return the studentId
	 */
	public int getStudentId() {
		return studentId;
	}

	/**
	 * @param studentId the studentId to set
	 */
	public void setStudentId(int studentId) {
		this.studentId = studentId;
	}

	/**
	 * @return the paymentAmount
	 */
	public int getPaymentAmount() {
		return paymentAmount;
	}

	/**
	 * @param paymentAmount the paymentAmount to set
	 */
	public void setPaymentAmount(int paymentAmount) {
		this.paymentAmount = paymentAmount;
	}

	/**
	 * @return the dueDate
	 */
	public LocalDate getDueDate() {
		return dueDate;
	}

	/**
	 * @param dueDate the dueDate to set
	 */
	public void setDueDate(LocalDate dueDate) {
		this.dueDate = dueDate;
	}

	/**
	 * @return the semester
	 */
	public String getSemester() {
		return semester;
	}

	/**
	 * @param semester the semester to set
	 */
	public void setSemester(String semester) {
		this.semester = semester;
	}
}

File: CRS-GROUP-A-Angular-UI-UX-App/src/app/models/payment.ts
export class Payment {
	
	private paymentId: number;
	private studentId: number;
	private paymentAmount: number;
	private dueDate: Date;
	private semester: string;
	private isPaid: number;
	private paymentMethod: string;
	
	constructor(
			paymentId: number, 
			paymentAmount: number,
			studentId: number, 
			dueDate: Date, 
			semester: string, 
			paymentMethod: string,
			isPaid: number) {

		this.paymentId = paymentId;
		this.studentId = studentId;
		this.paymentAmount = paymentAmount;
		this.dueDate = dueDate;
		this.semester = semester;
		this.paymentMethod = paymentMethod;
		this.isPaid = isPaid;
	}
	
	getIsPaid(): number {
		return this.isPaid;
	}

	setIsPaid(isPaid: number) {
		this.isPaid = isPaid;
	}

	getPaymentMethod(): string {
		return this.paymentMethod;
	}

    setPaymentMethod(paymentMethod: string) {
		this.paymentMethod = paymentMethod;
	}

	getPaymentId(): number {
		return this.paymentId;
	}

	setPaymentId(paymentId: number) {
		this.paymentId = paymentId;
	}

	getStudentId(): number {
		return this.studentId;
	}

	setStudentId(studentId: number) {
		this.studentId = studentId;
	}

	getPaymentAmount(): number {
		return this.paymentAmount;
	}

	setPaymentAmount(paymentAmount: number) {
		this.paymentAmount = paymentAmount;
	}

	getDueDate(): Date {
		return this.dueDate;
	}

	setDueDate(dueDate: Date) {
		this.dueDate = dueDate;
	}

	getSemester(): string {
		return this.semester;
	}

	setSemester(semester: string) {
		this.semester = semester;
	}
}